README.md

```diff
@@ -1,3 +1,56 @@
-# Backend app
+# Music Library
+
+This is an app designed to perform CRUD requests on a locally-hosted PostgreSQL database. 
+
+## Installation
+
+Pull and run a docker image of PostgreSQL:
+
+`docker run --name postgres -p 5432:5432 -e POSTGRES_PASSWORD=password -d postgres`
+
+Install the project to your machine:
+
+git clone `https://github.com/JordsCodes/music-library`
+
+
+## Usage
+
+Navigate to the root directory in your terminal and run:
+
+`npm start`
+
+Now that the project is running, we can make CRUD requests to the routes specified in the routes files for artists and albums. 
+
+This will update the locally-hosted Artists and Albums tables.
+
+For example, the user could make a POST request to http://localhost:3000/artists/.
+
+The body of the request should be written in JSON:
+
+`{ 
+"name": "Oasis"
+ "genre": "Rock"
+}`
+
+The user can also make GET, PUT, PATCH, and DELETE requests to routes `/artists`, `/artists/id/albums` and `/albums`. 
```

Comment:
Good that you have said what is available but I think it could be a little more clear which routes work with which request type. Particularly explaining the post to albums clearly as it's the most complex one, include the req.body like you have above for post to artists